config/jenkins-release.yaml

```diff
@@ -479,10 +479,12 @@ controller:
           authorizationStrategy:
             globalMatrix:
               permissions:
-                - "Overall/Administer:release-core"
-                - "Overall/SystemRead:authenticated"
-                - "Overall/Read:authenticated"
-                - "Job/Read:authenticated"
+                - "GROUP:Overall/Administer:release-core"
+                - "GROUP:Overall/Administer:admins"
```

Comment:
@daniel-beck @timja in common with @dduportal we've added this permission for our "admin" group on release.ci.jenkins.io, as we have already admin rights on other services.
OK for you?

Reply:
This isn't cert.ci so up to infra team 🤷